app/views/shared/_message.html.haml

```diff
@@ -0,0 +1,14 @@
+-# TODO: 分岐が汚すぎるからどうにかする
+- if (defined?(current_mentor) && message.user.type_id == 'mentor') || (defined?(current_student) && current_student&.id == message.user_id) || defined?(right)
```

Comment:
__[HAML-Lint]__

Line is too long. [159/80]


<!-- Comment by [SideCI](https://sideci.com) -->